examples/run_setup_tutorial.py

```diff
@@ -0,0 +1,100 @@
+"""
+=========
+Run Setup
+=========
+
+By: Jan N. van Rijn
+
+One of the key features of the openml-python library is that is allows to
+reinstantiate flows with hyperparameter settings that were uploaded before.
+This tutorial uses the concept of setups. Although setups are not extensively
+described in the OpenML documentation (because most users will not directly
+use them)
+
+A key requirement for reinstantiating a flow is to have the same scikit-learn
+version as the flow that was uploaded. This tutorial will upload the flow
+itself, so it can be ran with any scikit-learn version that is supported by
+this library.
+
+In this tutotial we will
```

Comment:
tutotial -> tutorial

Also, could you please add a colon at the end of the line?